definitions/tools/sequence_to_fastq_rna.cwl

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Picard: BAM to FASTQ"
+baseCommand: ["/bin/bash","makefastqs.sh"]
+requirements:
+    - class: ResourceRequirement
+      coresMin: 1
+      ramMin: 16000
+      tmpdirMin: 25000
+    - class: DockerRequirement
+      dockerPull: "mgibio/rnaseq:1.0.0"
+
+    - class: SchemaDefRequirement
+      types:
+          - $import: ../types/sequence_data.yml
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'makefastqs.sh'
+        entry: |
+            set -o pipefail
+            set -o errexit
+            set -o nounset
+            while getopts ":b:?1:?2:d:?" opt; do
+                case "$opt" in
+                    b)
+                        MODE=bam
+                        BAM="$OPTARG"
+                        ;;
+                    1)
+                        MODE=fastq
+                        FASTQ1="$OPTARG"
+                        ;;
+                    2)  
+                        MODE=fastq
+                        FASTQ2="$OPTARG"
+                        ;;
+                    d)
+                        OUTDIR="$OPTARG"
+                        ;;
+                esac
+            done
+
+            if [[ "$MODE" == 'fastq' ]]; then #must be fastq input
+                cp $FASTQ1 $OUTDIR/read1.fastq
+                cp $FASTQ2 $OUTDIR/read2.fastq
```

Comment:
Do we need to care if these FASTQs are gzipped?  I see the commented-out `gunzip` below; I was mostly wondering if we should at least preserve the file extension if it's a `fastq.gz` coming in.

Reply:
Added in a flag in the latest set of commits that attempts to unzip when possible and copies otherwise, while preserving the `.gz` extension if the file is in fact gzipped.